pyobjus/dylib_manager.py

```diff
@@ -1,9 +1,12 @@
 import os
 import ctypes
 import pyobjus
-from subprocess import call
-from objc_py_types import enum
-from debug import dprint
+try:
+    from subprocess import call
+except:
```

Comment:
because it depends of the compilation, i have no idea what will be the exception if something is not working right now

Reply:
and it's for iOS where subprocess is problematic, it has no impact on others

Reply:
When I don't know I usually prefer to let it crash eventually so that I know and I can be more specific.
Because what if this actually never fails, we have try/except in case and it never break?